### DISCUSSION TITLE
High cpu usage while viewing commits on github ui?

### DISCUSSION THREAD

**Discussion by martin-juul** [community/community#120048] [Repositories]
### Select Topic Area

Bug

### Body

When i visist https://github.com/angular/angular/compare/17.3.x...main in Chrome 124.0.6367.61 (Official Build) (64-bit) (cohort: M124 Rollout)  the tab starts hugging memory and cpu until it dies. In incognito without extensions and in a normal instance.

**Comment by LiteBrite82** [comment]
I was unable to replicate. Is it still doing it? 
I did notice some seriously poor performance though...  it seems to come and go....

What does the console say? 
Virus scanner?
VPN software like Nord?

**Reply by LiteBrite82** [reply]
That being said....

This community is managed by GitHub employees, but it is not a support forum and there is no SLA or guarantee each post will get a response from a community manager or product leader.

If your discussion provided product feedback or a feature request we encourage you to review the "Making Suggestions" [guidelines](https://github.com/community/community#making-suggestions) in the repo readme including: "Once you kick off a discussion, the GitHub product team will evaluate the feedback but will not be able to respond to every submission."

Thank you for contributing and continuing to support other users in the GitHub Community Discussions.

**Comment by martin-juul** [comment]
Chrome task manager

![image](https://github.com/community/community/assets/11337105/339f8459-49e6-41ef-806f-075a6f19c6d0)

Console:

![image](https://github.com/community/community/assets/11337105/bd536337-abe4-4305-95c5-fe59783c8b19)

I'm running plain Windows 11 with microsoft antivirus, no vpn software.

**Reply by LiteBrite82** [reply]
My top return on a google serarch for the error log was a known recent issue with 1password: https://1password.community/discussion/145003/console-error-retrieving-b5x-stateful-inline-icon-flag-errored-timed-out-falling-back

any possibilities there?

I don't see how that could cause a loop though.... assuming the resource hogging is a loop...

**Reply by LiteBrite82** [reply]
@martin-juul 
any updates?

**Reply by martin-juul** [reply]
Hi @LiteBrite82 thank you for checking in. This issue only seems to be popping up on my windows machines, but not on my personal mac. My work computer is running Windows 11, latest build. It seems to be unique to my work machine, which is not the most powerful machine (it produces a passmark score of about 9000). I'm back at work on Tuesday, if you want to catch up. I'm happy to show you on a screenshare if you want to see it in action?

**Reply by LiteBrite82** [reply]
This one is a shot in the dark and likely not the issue as you are running win 11. But hey, It couldn't hurt to check. I had to update / downgrade the package on my Mac as per my IT security dept. 
  
do you use node? do you have "xz" installed? 
This vulnerability [YouTube quick rundown](https://www.youtube.com/watch?v=jqjtNDtbDNI) was found about the same time you posted; skip to 8:27 for the description of the symptoms.

**Comment by myxoh** [comment]
Shot in the dark - but I'm seeing the exact same issue - on Chrome: Version 129.0.6668.70 (Official Build) (64-bit)
Windows 11 too...

**Comment by myxoh** [comment]
For additional context: I've removed all extensions, tried it in two different chrome sessions and reproduced the issue (not just on this PR but pretty much every >60 files changed PR) - I've got a pretty beefy i9 from 2023 too but making the UI pretty unusable.

Don't seem to be seeing this issue on Firefox
